k8s/search_result.go

```diff
@@ -64,23 +64,22 @@ func convertToStruct(obj unstructured.Unstructured) starlark.Value {
 
 func convertToStarlarkPrimitive(input interface{}) starlark.Value {
 	var value starlark.Value
-	switch input.(type) {
+	switch input := input.(type) {
```

Comment:
Use a different variable name other than `input` here since the iincoming arg is named input.